mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-warn_unused_ignores = True
+warn_unused_ignores = False
```

Comment:
probably should revert this

Reply:
oh I forgot to push this change. updated.